apps/cli/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import yargs from "yargs/yargs";
```

Comment:
need to init globals

- require network env var to be set
- set explorer/node based on that
- how do we want to load donera address? can i load the `@donera/dapp` module and get the address from there? What if i change the value to a donera instance instead?